rules/share-common-types-and-logic/rule.md

```diff
@@ -1,6 +1,6 @@
 ---
 type: rule
-title: Do you share common types and logic?
+title: Blazor - Do you share common types and logic?
```

Comment:
Bad content design - change to "Do you share common types and logic in Blazor?"

Reply:
Adding @pierssinclairssw 

Jayden - I disagree as I generally like prefixes.
See https://www.ssw.com.au/rules/prefixes/
-a

Reply:
@adamcogan  ⚠️ For the record I've downvoted that prefixes rule, please see the email thread "Re: Clean Architecture Rules - Add prefix" for some specific feedback from Tiago and I on this topic.